devicetypes/smartthings/leaksmart-water-sensor.src/leaksmart-water-sensor.groovy

```diff
@@ -0,0 +1,136 @@
+			state "battery", label:'${currentValue}% battery', unit:""
+		}
+		standardTile("refresh", "device.refresh", inactiveLabel: false, decoration: "flat", width: 2, height: 2) {
+			state "default", action: "refresh.refresh", icon: "st.secondary.refresh"
+		}
+		main "water"
+		details(["water", "temperature", "battery", "refresh"])
+	}
+}
+
+private getBATTERY_PERCENTAGE_REMAINING() { 0x0021 }
+private getTEMPERATURE_MEASURE_VALUE() { 0x0000 }
+private getEVENTS_ALERTS_CLUSTER() { 0x0B02 }
+
+def installed() {
+	sendEvent(name: "water", value: "dry", displayed: false)
+	refresh()
+}
+
+def parse(String description) {
+	def map = zigbee.getEvent(description)
+	if(!map) {
+		map = parseAttrMessage(description)
+	} else if (map.name == "temperature") {
+		if (tempOffset) {
+			map.value = (int) map.value + (int) tempOffset
+		}
+		map.descriptionText = temperatureScale == 'C' ? "${device.displayName} was ${map.value}°C" : "${device.displayName} was ${map.value}°F"
+		map.translatable = true
+	}
+
+	def result = map ? createEvent(map) : [:]
+
+	if (description?.startsWith('enroll request')) {
+		def cmds = zigbee.enrollResponse()
+		result = cmds?.collect { new physicalgraph.device.HubAction(it)}
+	}
+	log.debug "Description ${description} parsed to ${result}"
+	return result
+}
+
+private Map parseAttrMessage(description) {
+	def descMap = zigbee.parseDescriptionAsMap(description)
+	def map = [:]
+	if(descMap?.clusterInt == EVENTS_ALERTS_CLUSTER) {
+		map = descMap?.data[1] == "81" ? getWaterDetection(descMap?.data[2]) : [:]
+	} else if(descMap?.clusterInt == zigbee.TEMPERATURE_MEASUREMENT_CLUSTER && descMap.commandInt == 0x07) {
+		if (descMap.data[0] == "00") {
+			sendCheckIntervalEvent()
+		} else {
+			log.warn "TEMP REPORTING CONFIG FAILED - error code: ${descMap.data[0]}"
+		}
+	}
+	return map
+}
+
+private Map getWaterDetection(alertData) {
+	def value = (alertData == "11") ? "wet" : "dry"
+	def description = (value == "wet") ? "detected" : "not detected"
+	def result = [name: "water", value: value, descriptionText: "Water was ${description}", displayed: true, isStateChanged: true]
+	return result
+}
+
+private sendCheckIntervalEvent() {
+	sendEvent(name: "checkInterval", value: 2 * 60 * 60 + 2 * 60, displayed: false, data: [protocol: "zigbee", hubHardwareId: device.hub.hardwareID, offlinePingable: "1"])
+}
+
+def ping() {
+	refresh()
+}
+
+def refresh() {
+	return zigbee.readAttribute(zigbee.POWER_CONFIGURATION_CLUSTER, BATTERY_PERCENTAGE_REMAINING) +
+			zigbee.readAttribute(zigbee.TEMPERATURE_MEASUREMENT_CLUSTER, TEMPERATURE_MEASURE_VALUE)
+}
+
+def configure() {
+	sendCheckIntervalEvent()
+	log.debug "Configuring Reporting"
+	def configCmds = zigbee.configureReporting(zigbee.POWER_CONFIGURATION_CLUSTER, BATTERY_PERCENTAGE_REMAINING, DataType.UINT8, 30, 21600, 0x01) +
+			zigbee.temperatureConfig(30, 300) +
+			zigbee.configureReporting(EVENTS_ALERTS_CLUSTER, 0x0000, 0x00, 5, 14400, null)
```

Comment:
This cluster doesn't appear to have any attributes, so I'm unsure if this works.

Reply:
Without this configuration device doesn't report any status change.